### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by JasonBristol**
Fixes for #35 & #38, in addition to possible fix for #37

### CODE REVIEW

File: wrapper/core/events.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def __iter__(self):
 
     def callEvent(self, event, payload):
         if event == "player.runCommand":
+            print payload["args"]
```
Comment (suresttexas00) [inline wrapper/core/events.py]: This event is not actually used... at least it is not supposed to accept a returned payload.. and it is not to be used to parse/register commands.  The two events I use are player.chatbox and player.rawMessage.

File: wrapper/core/commands.py [hunk 2]
```diff
@@ -72,6 +72,7 @@ def playerCommand(self, payload):
             buildString = self.wrapper.getBuildString()
             if len(get_args(payload["args"], 0)) > 0:
                 subcommand = get_args(payload["args"], 0)
+                print subcommand
```
Comment (suresttexas00) [inline wrapper/core/commands.py]: I think these print statements should come back out now that these issues are resolved (for now).

File: wrapper/core/events.py (hunk 1 above)
Comment (JasonBristol) [inline wrapper/core/events.py]: These were debugging statements, my next PR will remove them

File: wrapper/core/commands.py (hunk 2 above)
Comment (JasonBristol) [inline wrapper/core/commands.py]: Have confidence ;) Wrapper has always been buggy, these bugs are at least minor, many major ones have been squashed

**Pull request merged**